README.adoc

```diff
@@ -101,58 +104,57 @@ In the following example, if there are more than three very high severity vulner
 [source,yaml]
 ----
 
-     - name: Get sourcecode
+      - name: Check out source code
         uses: cloudbees-io/checkout@v1
 
-      - name: Run TruffleHog Secret SAST
+      - name: Run TruffleHog secret scan with threshold
         uses: cloudbees-io/trufflehog-secret-scan-code@v1
         with:
           branch: ${{ cloudbees.scm.branch }}
           threshold-very-high: 3
 ----
 
-== Advanced Usage examples
-
-In the following example, local codebase is scanned from the specified commit hash.
+Scan a local codebase branch starting from a specified commit hash:
 
 [source,yaml]
 ----
 
-     - name: Get sourcecode
+      - name: Check out source code
```

Comment:
Can you please update this example like below (While scanning a local codebase starting from specific commit hash, codebase is expected to be checked out with full history).
```
      - name: Check out source code
        uses: cloudbees-io/checkout@v1
        with:
          fetch-depth: 0 # Fetch the full history

      - name: Run TruffleHog starting from commit
        uses: cloudbees-io/trufflehog-secret-scan-code@v1
        with:
          branch: ${{ cloudbees.scm.branch }}
          since-commit: ${{ commit.hash }}
          threshold-very-high: 3
 ```